app/models/case/overturned_ico/foi.rb

```diff
@@ -1,7 +1,8 @@
 class Case::OverturnedICO::FOI < Case::OverturnedICO::Base
 
   def self.type_abbreviation
-    'OVERTURNED_FOI '
+    'OVERTURNED_FOI'
```

Comment:
When we do the Overturned FOIs.  This PR is for SARs only.

Reply:
But if we already have this method shouldn't we test it otherwise we risk just ignoring it. Having had a look through `spec/models` it looks like we only test  type_abbreviation for FOI and OVERTURNED_SAR. We are not tests original SAR or ICO Appeals.